oval_graph/oval_tree/evaluate.py

```diff
@@ -44,10 +68,21 @@ def oval_operator_one(result):
 
 
 def oval_operator_or(result):
-    """
-        The OR operator produces a true result if one or more arguments is true. If every argument
-        is false, the result of the OR is false. If one or more of the arguments are unknown and
-        if none of arguments are true, then the OR operator produces a result of unknown.
+    """The OR operator produces a true result if one or more arguments is true. If every argument
+       is false, the result of the OR is false. If one or more of the arguments are unknown and
+       if none of arguments are true, then the OR operator produces a result of unknown.
+
+    Args:
+            result (dict): Dictionary representing frequencies of values
+
+    Returns:
+            str. return values::
+
+                true
+                false
+                error
+                unknown
+                noteval
     """
     out_result = None
     if greater_zero(result, 'true_cnt'):
```

Comment:
`true_cnt` etc. are not good names, it's better to use at least `true_count` or `num_of_true` (num is an established abbreviation, cnt is not).